core/adapters/json_parse.go

```diff
@@ -82,37 +83,19 @@ func dig(js *simplejson.Json, path []string) (*simplejson.Json, error) {
 
 // only error if any keys prior to the last one in the path are nonexistent.
 // i.e. Path = ["errorIfNonExistent", "nullIfNonExistent"]
-func moldErrorOutput(js *simplejson.Json, path []string, input models.RunInput) models.RunOutput {
-	if _, err := getEarlyPath(js, path); err != nil {
+func moldErrorOutput(js gjson.Result, path []string) models.RunOutput {
+	if _, err := dig(js, path[:len(path) - 1]); err != nil {
 		return models.NewRunOutputError(err)
 	}
 	return models.NewRunOutputCompleteWithResult(nil)
 }
 
-func getEarlyPath(js *simplejson.Json, path []string) (*simplejson.Json, error) {
```

Comment:
this function is a copy of `dig()` with the different that the input path is a sub-slice. I refactored to use dig directly.